records_mover/records/types.py

```diff
@@ -105,16 +105,29 @@ class UrlDetailsEntry(TypedDict):
 HintQuoteChar = str
 
 
-if TYPE_CHECKING:
-    BootstrappingRecordsHints = TypedDict('BootstrappingRecordsHints',
-                                          {
-                                              'quoting': HintQuoting,
-                                              'header-row': HintHeaderRow,
-                                              'field-delimiter': HintFieldDelimiter,
-                                              'encoding': HintEncoding,
-                                              'escape': HintEscape,
-                                              'compression': HintCompression,
-                                          },
-                                          total=False)
-else:
-    BootstrappingRecordsHints = RecordsHints
+BootstrappingRecordsHints = TypedDict('BootstrappingRecordsHints',
+                                      {
+                                          'quoting': HintQuoting,
+                                          'header-row': HintHeaderRow,
+                                          'field-delimiter': HintFieldDelimiter,
+                                          'encoding': HintEncoding,
+                                          'escape': HintEscape,
+                                          'compression': HintCompression,
+                                      },
+                                      total=False)
+
+
+# TODO: these should live in hints.py - existing stuff should probably move
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker